docs/ftrl.rst

```diff
@@ -124,11 +122,27 @@ can be accessed as
 ::
 
   fi = ftrl_model.feature_importances
-  
+
 where ``fi`` will be a frame of shape ``(nfeatures, 2)`` containing
 feature names and their importances, that are normalized to [0; 1] range.
 
 
+Feature Interactions
+-------------------
+
+By default each column of a training dataset is considered as a feature
+by FTRL model. User can provide additional features by specifying
+a list of feature interactions, for instance as
+
+::
+
+  ftrl_model.interactions = [["C0", "C1", "C3"], ["C2", "C5"]]
+
+where ``C*`` are column names from a training dataset. `interactions`
+should be set before a call to `fit()` method, and can not be changed
+once the model is trained.
```

Comment:
Not entirely clear from this doc what kind of interactions are created: is this 2 interactions `C0*C1*C3` and `C2*C5`; or 6 interactions `C0*C2, C0*C5, C1*C2, C1*C5, C3*C2, C3*C5` ?

Reply:
Thanks, clarified this part. May need to work on a more detailed manual at some point.